litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts

```diff
@@ -116,13 +113,9 @@ export default [
       'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/pod-delete/workflow.yaml',
     chaosWkfCRDLink_Recur:
       'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/pod-delete/workflow_cron.yaml',
-
```

Comment:
Why is the git link and isCustom parameters removed? @ishangupta-ds

Reply:
Added them back @Jonsy13